Modules/VAMI/VAMI.psm1

```diff
@@ -621,53 +652,150 @@ Function New-VAMIUser {
      Organization:  VMware
      Blog:          www.virtuallyghetto.com
      Twitter:       @lamw
-	===========================================================================
-	.SYNOPSIS
-		This function to create new VAMI local user using VAMI interface (5480)
+     Modifed by:    Michael Dunsdon
+     Twitter:      @MJDunsdon
+     Date:         September 16, 2020
+    ===========================================================================
+    .SYNOPSIS
+        This function to create new VAMI local user using VAMI interface (5480)
         for a VCSA node which can be an Embedded VCSA, External PSC or External VCSA.
-	.DESCRIPTION
-		Function to create a new VAMI local user
-	.EXAMPLE
+    .DESCRIPTION
+        Function to create a new VAMI local user
+    .EXAMPLE
         Connect-CisServer -Server 192.168.1.51 -User administrator@vsphere.local -Password VMware1!
-        New-VAMIUser -name lamw -fullname "William Lam" -role "operator" -email "lamw@virtuallyghetto.com" -password "VMware1!"
+        New-VAMIUser -name lamw -fullname "William Lam" -role "operator" -email "lamw@virtuallyghetto.com" -password "VMware1!" -passwordexpires  -passwordexpiresat "1/1/1970" -maxpasswordage 90
+    .NOTES
+        Modified script to account for Newer VCSA. Script supports 6.5 and 6.7 VCSAs.
+        Also added new Parameters to script.
 #>
     param(
-        [Parameter(
-            Mandatory=$true)
-        ]
+        [Parameter(Mandatory=$true)]
         [String]$name,
-        [Parameter(
-            Mandatory=$true)
-        ]
+        [Parameter(Mandatory=$true)]
         [String]$fullname,
-        [Parameter(
-            Mandatory=$true)
-        ]
-        [ValidateSet("admin","operator","superAdmin")][String]$role,
-        [Parameter(
-            Mandatory=$false)
-        ]
+        [Parameter(Mandatory=$true)]
+        [ValidateSet("admin","operator","superAdmin")]
+        [String]$role,

```

Comment:
PowerShell parameters and veriable names is good to be CamelCase. PowerShell is case insensitive but to make it look real PowerShell Function it will be good to make them CamelCase.

https://powershell.org/forums/topic/parameter-and-variable-naming-camelcase/